### DISCUSSION TITLE
Reading data from stdin?

### DISCUSSION THREAD

**Discussion by 097115** [andmarti1424/sc-im#771] [General]
Just wanted to clarify, there's no way (and no smart trick) to pipe some data from stdin to scim? We have to create an intermediary temp file for that, right?

**Comment by andmarti1424** [comment]
False. You can do something like this:
`echo LET A0=2 | ./sc-im`

**Comment by 097115** [comment]
Yeah, I saw that in help :)

However, piping something like `curl https://eodhistoricaldata.com/api/eod/AAPL.US?api_token=demo` is not so straightforward, right?

May be with some awk in-between... But then again, it's probably easier to create a temp file.
